### PR TITLE
Added Norm function 

### LINKED ISSUE #11936: norm

### LINKED ISSUE #17223: norm

### PR DESCRIPTION
**Pull request by avi-thakur-0812**
Close #11936

### CODE REVIEW

**Comment by avi-thakur-0812** [comment]
@Daniel4078 Hey! It's my first time contributing, could you please let me know why do I keep getting the lint failing test , and also specify any changes that need to be made. 

Thanks a lot!

File: ivy/functional/backends/torch/.cph/.linear_algebra.py_b854ef339ed8693b58e3457d6ddd99a3.prob [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"Local: linear_algebra","url":"d:\\Ivy\\ivy\\ivy\\functional\\backends\\torch\\linear_algebra.py","tests":[],"interactive":false,"memoryLimit":1024,"timeLimit":3000,"srcPath":"d:\\Ivy\\ivy\\ivy\\functional\\backends\\torch\\linear_algebra.py","group":"local","local":true}
```
Comment (Daniel4078) [inline ivy/functional/backends/torch/.cph/.linear_algebra.py_b854ef339ed8693b58e3457d6ddd99a3.prob:1]: you should not push this kind of hidden local files to git.

File: ivy/functional/backends/torch/linear_algebra.py [hunk 2]
```diff
@@ -509,10 +509,15 @@ def vector_norm(
 vector_norm.support_native_out = True
 
 
+
```
Comment (Daniel4078) [inline ivy/functional/backends/torch/linear_algebra.py:512]: you should only keep two empty lines between each chunk of code, you can imitate other similar files for the correct format.

File: ivy/functional/backends/torch/linear_algebra.py [hunk 3]
```diff
@@ -509,10 +509,15 @@ def vector_norm(
...
 # Extra #
 # ----- #
 
 
+
```
Comment (Daniel4078) [inline ivy/functional/backends/torch/linear_algebra.py:517]: like all these four extra empty lines are not needed

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 4]
```diff
@@ -74,6 +74,47 @@ def _get_dtype_and_matrix(draw):
         )
     )
 
+#norm
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:77]: comment also count as code, so keep two empty lines before it. also, you need a space between # and norm

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 5]
```diff
@@ -74,6 +74,47 @@ def _get_dtype_and_matrix(draw):
...
+@handle_frontend_test(
+    fn_tree="torch.linalg.norm",
+    dtype_values_axis=helpers.dtype_and_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        valid_axis=True,
+
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:83]: please remove this unnecessary empty line

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, I have checked the log of the lint test and here are my recommended changes.

**Comment by Daniel4078** [comment]
add_frontend_checklist

**Comment by Daniel4078** [comment]
I found that this pull request https://github.com/unifyai/ivy/pull/16729 is also dealing with this task. It seems like there are some misunderstanding here.

**Pull request closed without merging**